db/record.c

```diff
@@ -317,9 +316,7 @@ int add_record(struct ireq *iq, void *trans, const uint8_t *p_buf_tag_name,
     }
 
     struct schema *ondisktagsc; // schema for .ONDISK
-    int tag_same_as_ondisktag = (strcmp(tag, ondisktag) == 0);
-
-    if (tag_same_as_ondisktag) {
+    if (strcmp(tag, ondisktag) == 0) {
```

Comment:
same check was done above in line 252. You can cache this check at line 252 and use here.

Reply:
Done. Thank you!